File-Formats/DocIO/Working-with-Tables.md

```diff
@@ -2247,6 +2247,373 @@ document.Close();
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Tables/Apply-custom-table-style).
 
+### Apply Base Style
+
+Table styles can be based on other table styles also. When applying base style, the new style will inherit the values of the base style that are not explicitly redefined in the new style. You can apply custom table style or built-in table as a base for the table style using BuiltinTableStyle API.
+
+The following code example illustrates how to apply built-in and custom table styles as base style for another custom table.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Create a new Word document.
+using (WordDocument document = new WordDocument())
+{
+    //Add one section and paragraph in document.
+    document.EnsureMinimal();
+    //Add a table to the Word document.
+    WTable table = document.LastSection.AddTable() as WTable;
+    table.ResetCells(3, 2);
+    table[0, 0].AddParagraph().AppendText("Row 1 Cell 1");
+    table[0, 1].AddParagraph().AppendText("Row 1 Cell 2");
+    table[1, 0].AddParagraph().AppendText("Row 2 Cell 1");
+    table[1, 1].AddParagraph().AppendText("Row 2 Cell 2");
+    table[2, 0].AddParagraph().AppendText("Row 3 Cell 1");
+    table[2, 1].AddParagraph().AppendText("Row 3 Cell2");
+    //Add a new custom table style.
+    WTableStyle tableStyle = document.AddTableStyle("CustomStyle1") as WTableStyle;
+    tableStyle.TableProperties.RowStripe = 1;
+    //Apply conditional formatting for first row.
```

Comment:
//Apply conditional formatting for the first row.

Reply:
Addressed